### DISCUSSION TITLE
Stripe Android SDK Bindings

### DISCUSSION THREAD

**Discussion by BrandanN21** [dotnet/maui#23282] [Q&A]
My team and I have been working to implement Google Pay with the Stripe Android SDK utilizing the Android Binding Library process. We have run into many roadblocks for this process. We have been able to get our .jar files and import them into the library project but run into 500+ errors with the auto generated C# bindings. There is documentation to use the metadata.xml to update accordingly, which we have tried to do but it doesn't resolve the issues. 

I'm curious if a) anyone has had any luck getting this SDK working with a MAUI application or b) is there a better way to resolve issues with the auto generated bindings?

It seems counterintuitive to have to redo all of the generated bindings. Are there plans to provide more robust documentation for this process, or who/where to reach out for help. It doesn't appear there are many resources that can provided assistance getting through this process. 

Thanks, appreciate any ideas or suggestions anyone has!

**Comment by pictos** [comment]
You can try the slim binding approach, it's useful if you don't need to access most of the APIs on the sdk, here's a talk that may provide you some info
https://www.youtube.com/watch?v=bgK_6anwMcw
